### ISSUE TITLE
`use` has no effect when jabba installed via homebrew

### ISSUE THREAD

**Issue by wimdavies** [Jabba-Team/jabba#46]
Similar situation to #37. 
I originally installed jabba via homebrew. I could successfully `jabba install` multiple java versions, but `jabba use` had no effect on JAVA_HOME and the output of `jabba current` was nothing.

After running `brew uninstall jabba` and re-installing Jabba directly via curl, `jabba use` now successfully sets an installed java version.

**Comment by patrick-mccourt** [comment]
Hi @wimdavies can I just check, is this homebrew on Mac OS? If so which version of MacOS

**Comment by wimdavies** [comment]
> Hi @wimdavies can I just check, is this homebrew on Mac OS? If so which version of MacOS

Sorry for neglecting to include this information.
Yes, this is homebrew on macOS, Sonoma 14.6.1.

**Comment by patrick-mccourt** [comment]
Still need some time to look into this better, just out of curiosity, could you try this older version:

`brew install jabba@0.12.1` 

and let me know if it works

**Comment by patrick-mccourt** [comment]
Hi @wimdavies, the homebrew installer works differently to the install script, and so misses some of shell integration steps the script does, can you try adding: 

[ -s "$HOMEBREW_PREFIX/opt/jabba/jabba.sh" ] && . "$HOMEBREW_PREFIX/opt/jabba/jabba.sh"

to your .zshrc on Mac (assuming you've not changed your shell from the default zsh)

t

**Comment by iansedano** [comment]
> can you try adding:
>
> `[ -s "$HOMEBREW_PREFIX/opt/jabba/jabba.sh" ] && . "$HOMEBREW_PREFIX/opt/jabba/jabba.sh"`
>
> to your .zshrc on Mac (assuming you've not changed your shell from the default zsh)

I had the same problem as described here (on macOS Sequuoia 15.2) and adding that to my .zshrc file fixed it for me. I did have to reinstall the target version of Java. Once it was installed it was automatically set as `current`, however, I need to run `jabba use <VERSION>` on every new session.

**Comment by BetterB0y** [comment]
@patrick-mccourt I still have an issue with homebrew and use. Current is not showing anythig. Check is producing this output:
```
Jabba Installation Check
========================
JABBA_HOME:           /Users/Better/.jabba
JAVA_HOME:
Current Shell:        zsh
Integration File:     /Users/Better/.jabba/jabba.sh
Integration Exists:   ✗ No (run 'jabba init' to create)
RC File:              /Users/Better/.zshrc
RC File Configured:   ✓ Yes
Shell Integration:    ✗ Not active (jabba is binary only)

To activate shell integration, restart your shell or run:
  source "/Users/Better/.jabba/jabba.sh"
```

**Comment by patrick-mccourt** [comment]
Hi @BetterB0y,

Does this still persist after running jabba init and restarting your shell?

**Comment by BetterB0y** [comment]
@patrick-mccourt Yes. Issue still persist. Still after restarting i get that shell intergration is not active
